CHANGELOG.md

```diff
@@ -2,10 +2,6 @@
 
 This file is used to list changes made in each version of the chef_client_updater cookbook.
 
-## 3.5.2 (2019-01-30)
```

Comment:
@gholtiii Why are you removing the changelog entry?

Reply:
That was not intentional - not sure how that happened but it's fixed